src/js/jsx/Scrim.jsx

```diff
@@ -198,6 +230,11 @@ define(function (require, exports, module) {
             if (!nextState.activeDocumentInitialized || !nextState.recentFilesInitialized) {
                 return false;
             }
+            
+            if (this.state.isDragging === nextState.isDragging &&
```

Comment:
There is a bug here. If I'm not dragging at all... but transform changes, this is going to prevent us from re-rendering the scrim with correct transform.